saskatoon/sitebase/locale/fr/LC_MESSAGES/django.po

```diff
@@ -17,244 +17,769 @@ msgstr ""
+msgid "Shared"
+msgstr ""
+
+#: sitebase/templates/app/list_views/filters.html:14
+msgid "Filters"
+msgstr ""
+
+#: sitebase/templates/app/list_views/filters.html:27
+msgid "Filter"
+msgstr ""
+
+#: sitebase/templates/app/list_views/filters.html:32
+msgid "Reset"
+msgstr "Réinitialiser"
+
+#: sitebase/templates/app/list_views/harvest/data_row.html:10
+msgid "Are you really sure you want to delete this harvest?"
 msgstr ""
 
-#: sitebase/templates/app/property_list.html:95
+#: sitebase/templates/app/list_views/harvest/view.html:14
+msgid "Leader"
+msgstr ""
+
+#: sitebase/templates/app/list_views/property/data_row.html:11
+msgid "Are you really sure you want to delete this property?"
+msgstr ""
+
+#: sitebase/templates/app/list_views/property/status.html:7
+msgid "PENDING"
+msgstr ""
+
+#: sitebase/templates/app/list_views/property/status.html:9
 msgid "validated"
 msgstr ""
 
-#: sitebase/templates/app/property_list.html:98
+#: sitebase/templates/app/list_views/property/status.html:12
 msgid "not yet authorized"
 msgstr ""
 
-#: sitebase/templates/app/property_list.html:100
+#: sitebase/templates/app/list_views/property/status.html:14
 msgid "authorized"
 msgstr ""
 
-#: sitebase/templates/app/property_list.html:102
+#: sitebase/templates/app/list_views/property/status.html:16
 msgid "unauthorized"
 msgstr ""
 
-#: sitebase/templates/app/property_list.html:105
+#: sitebase/templates/app/list_views/property/status.html:19
 msgid "inactive"
 msgstr ""
 
-#: sitebase/templates/registration/login.html:105
-msgid "Login"
+#: sitebase/templates/app/list_views/property/view.html:14
+msgid "Picks"
 msgstr ""
 
-#~ msgid "Volunteer Waiver"
-#~ msgstr "Formulaire de Décharge Bénévole"
+#: sitebase/templates/app/list_views/property/view.html:15
+msgid "Last pick"
+msgstr ""
 
-#~ msgid "Pre-pick Guide"
-#~ msgstr "Guide de Pré-Cueillette"
+#: sitebase/templates/app/participation_create.html:54
+msgid "Request to join this harvest"
+msgstr ""
 
-#~ msgid "Equipment Points (Annex B)"
-#~ msgstr "Points d'Equipement (Annexe B)"
+#: sitebase/templates/app/participation_create.html:77
+msgid ""
+"Sorry, this harvest is not open for requests. You can check the <a href='/"
+"calendar'>calendar</a> for other harvests."
+msgstr ""
 
-#~ msgid "Planning Harvests Using the Properties List (Annex D)"
-#~ msgstr "Planifier Récoltes en Utilisant la Liste des Proppriétés (Annexe D)"
+#: sitebase/templates/app/participation_create.html:90
+msgid "Sorry, this Harvest is not opened for requests"
+msgstr ""
 
-#~ msgid "Map: Equipment Points, Beneficiary Organizations & Community Fridges"
-#~ msgstr ""
-#~ "Carte: Points d'Equipement, Organismes Bénéficiaires & Frigos "
-#~ "Communautaires"
+#: sitebase/templates/app/participation_manage.html:68
+msgid "Update"
+msgstr ""
 
-#~ msgid "Welcome"
-#~ msgstr "Bienvenu(e)"
+#: sitebase/templates/registration/login.html:104
+msgid "Login"
+msgstr ""
 
-#~ msgid "Change password"
+#~ msgid "hehe"
```

Comment:
I'm confused, where is the "hehe" coming from?

Reply:
@2ynn , I accidentally pushed the test patch, I have updated the PR now. Please review.